common/src/main/resources/css/charts.css

```diff
@@ -0,0 +1,68 @@
+@import 'https://code.highcharts.com/css/highcharts.css';
```

Comment:
Will this be loaded at build time? or be included when loading the page?

Reply:
It's resolved at page load time.

I'd like to eventually extract the css from the facade artifacts and resolve at build time, but I'm not sure it's making it into the facade artifacts or we have to do something to include it there.